src/elm/DappInterface/CommonViews.elm

```diff
@@ -174,8 +174,10 @@ pageHeader userLanguage page connectedWallet account _ governanceState _ =
 
                         _ ->
                             emptyClasses
+                v2MarketsExternalLink = "https://app.compound.finance/markets?market=1_Compound+V2_0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B"
```

Comment:
We really need short links 😆

Reply:
yea definitely, gonna try to see how much effort it is, and if I can timebox a quick solution